pr-data.csv

```diff
@@ -1379,8 +1379,10 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/sql-engine,org.apache.ignite.internal.sql.engine.exec.MockedStructuresTest.testCreateTableMultiplePk,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/sql-engine,org.apache.ignite.internal.sql.engine.exec.MockedStructuresTest.testDropTable,ID,,,
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/sql-engine,org.apache.ignite.internal.sql.engine.StopCalciteModuleTest.testStopQueryOnNodeStop,ID,,,
+https://github.com/apache/ignite-3,35eda6eee8601d1c34a1cf5c074b2fa169bb4a9a,modules/storage-api,org.apache.ignite.internal.storage.AbstractMvPartitionStorageTest.abortOfInsertMakesRowNonExistentForReadByTimestamp,ID,,,
+https://github.com/apache/ignite-3,35eda6eee8601d1c34a1cf5c074b2fa169bb4a9a,modules/storage-api,org.apache.ignite.internal.storage.AbstractPartitionStorageTest.testGetAndReplaceClosureIfExistsFalse_entryNotExists,ID,,,
```

Comment:
Thanks for putting all your changes in one commit. I've just noticed here the class name is `Abstract...Test`. Can you run this test by itself, e.g., `mvn test -Dtest=Abstract...Test#test...`. Many times one can't run an abstract class (or more precisely, a test in an abstract class), and has to run some concrete class that extends this abstract class. We want to provide here the name of the test that can be run, not the name where the test code may be written.